File: Model/src/main/java/model/domain/Agency.java
package model.domain;

public class Agency implements Identifiable<String> {
    private String name;
    private String password;

    public Agency(String name, String password) {
        this.name = name;
        this.password = password;
    }

    @Override
    public String getId() {
        return name;
    }

    @Override
    public void setId(String id) {
        this.name = id;
    }

    public String getName() {
        return name;
    }

    public String getPassword() {
        return password;
    }

    @Override
    public String toString() {
        return "Agency: " + name;
    }
}

File: Model/Agency.cs
using System;

namespace Model
{
    [Serializable]
    public class Agency : IIdentifiable<string>
    {
        public string Name { get; set; }
        public string Password { get; set; }
        
        public Agency(string name, string password)
        {
            Name = name;
            Password = password;
        }

        public override string ToString()
        {
            return Name;
        }

        public string getId()
        {
            return Name;
        }

        public void setId(string id)
        {
            Name = id;
        }
    }
}
